index.html

```diff
@@ -7190,8 +7190,8 @@ <h2 id="AudioWorklet-Examples">
             </dt>
             <dd>
               <p>
-                An AudioBuffer where the output audio data MUST be written. 
-                 It will have a number of channels equal to the
+                An AudioBuffer where the output audio data MUST be written. It
+                will have a number of channels equal to the
```

Comment:
"It will" or "It MUST"?  Not really sure if we should bother to change this here.